### ISSUE TITLE
Shuffling+clustering round (best params) using classification

### ISSUE THREAD

**Issue by gcroci2** [DeepRank/3D-Vac#134]
After having determined the influence of standardization (#124), weighted loss function (#126), batch size (#127), and batch normalization (#131), experiment with clusters, using the choices that gave better results for shuffled data.
Common features among the following experiments:
- Data used are in data/pMHCI/features_output_folder/GNN/residue/230329/ (generated in #140 )
  - Residue-level queries
- Model is a slightly **improved naive GNN** (`PMHCI_Network01` in src/4_train_models/GNN/I/classification/struct/pmhc_gnn.py)
  - Added a convolutional and a linear layers
- **Standardization** applied to all features
- **Weighted loss function**
- **Batch size 64**
- `pssm` feature removed
- Cross entropy loss
- Adam optimizer
- **Number of epochs increased** (70 epochs, min_epoch 45, earlystop_patience 20, earlystop_maxgap 0.06)

Experiments:
- [x] Plot the best model obtained by using shuffled data, and use such parameters in all the below experiments
- [x] Clustering on peptides' clusters (cluster 3 is testset, see #113 for more details)
- [x] Clustering on alleles (cluster 1 is testset, see #113 for more details)
- [x] Only to confirm it doesn't work, clustering on alleles (HLA-C is test)
- [x] Compare sequence-based model and best CNN as baselines
- [x] Show improvements made from the very beginning (no standardization, no other parameters improvement), only for GNNs
